content/v2.0/reference/syntax/annotated-csv.md

```diff
@@ -150,25 +151,103 @@ Flux supports the following dialect options for `text/csv` format.
+To encode a table with its group key, the `datatype`, `group`, and `default` annotations must be included.
+If a table has no rows, the `default` annotation provides the group key values.
 {{% /note %}}
 
-##### Example
+## Data types
+
+| Datatype     | Flux type  | Description                                                                                                    |
+| :--------    | :--------- | :----------                                                                                                    |
+| boolean      | bool       | "true" or "false"                                                                                              |
+| unsignedLong | uint       | unsigned 64-bit integer                                                                                        |
+| long         | int        | signed 64-bit integer                                                                                          |
+| double       | float      | IEEE-754 64-bit floating-point number                                                                          |
+| string       | string     | UTF-8 encoded string                                                                                           |
+| base64Binary | bytes      | base64 encoded sequence of bytes as defined in RFC 4648                                                        |
+| dateTime     | time       | instant in time, may be followed with a colon : and a description of the format (number, RFC3339, RFC3339Nano) |
+| duration     | duration   | length of time represented as an unsigned 64-bit integer number of nanoseconds                                 |
+
+
+## Line protocol elements
+The `datatype` annotation also accepts line protocol elements.
+The [`influx write` command](/v2.0/reference/cli/influx/write/) uses the line protocol
+element to convert annotated CSV into line protocol when writing to InfluxDB.
+
+| Line protocol element | Description                                                     |
+|:--------------------- |:-----------                                                     |
+| `measurement`         | column value is the measurement                                 |
+| `field` _(default)_   | column header is the field key, column value is the field value |
+| `tag`                 | column header is the tag key, column value is the tag value     |
+| `time`                | column value is the timestamp _(alias for `dateTime`)_          |
+| `ignore` or`ignored`  | column is ignored and not included in line protocol             |
+
+### Mixing data types and line protocol elements
+Columns with [data types](#data-types) (other than `dataTime`) in the
+`#datatype` annotation are treated as **fields** when converted to line protocol.
+Columns without a specified data type default to `field` when converted to line protocol
+and **column values are left unmodified**.
```

Comment:
are lines 200-201 and 202-203 saying same?

If you mix columns .... in the `#datatype` annotation, the column headers (or only unspecified column headers?) are converted to **fields** in line protocol and column values...?

Reply:
No, they aren't the same. I've added a link to an example below that illustrates what happens.